apps/archive/archive_media.py

```diff
@@ -102,3 +114,24 @@ def get_file_from_document(self, doc):
             return content, content_type, decode_metadata(metadata)
 
         return file, file.content_type, file.metadata
+
+    def upload_file_to_video_server(self, doc):
+        """
+        Upload file to video server and create rendition for it
+        :param doc: info of file
+        :return:
+        """
+        file = doc.get('media')
+        # upload video to video server
+        res = self.videoEditor.post(doc.get('media'))
+        doc['media'] = res['_id']
+        metadata = res.get('metadata')
+        # create renditions
+        rend = {
+            'href': res.get('url'),
+            'media': res.get('_id'),
```

Comment:
It's better to use `video_editor_id` not `media` to store `_id` of project. It will give us an ability to know if this rendition belongs to video editor or not.